lib/src/presentation/views/home.dart

```diff
@@ -1,20 +1,321 @@
+import 'package:carousel_slider/carousel_slider.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:get/get.dart';
+import 'package:lottie/lottie.dart';
 
+import '../../core/utils/assets_constants.dart';
+import '../../core/utils/service_constants.dart';
 import '../../core/utils/string_constants.dart';
+import '../../core/utils/dimensions_constants.dart';
+import '../../data/models/recipe_model.dart';
+import '../controllers/recipes_controller.dart';
+import '../widgets/custom_app_bar.dart';
+import '../widgets/empty_screen.dart';
+import '../widgets/error_screen.dart';
+import '../widgets/tags_style.dart';
 
-class Home extends StatefulWidget {
-  const Home({Key? key}) : super(key: key);
+class HomePage extends StatelessWidget {
+  HomePage({Key? key}) : super(key: key);
 
-  @override
-  State<Home> createState() => _HomeState();
-}
+  final RecipesController recipesController = Get.find();
 
-class _HomeState extends State<Home> {
   @override
   Widget build(BuildContext context) {
-    return const Scaffold(
-      body: Center(
-        child: Text(StringConstants.homeTitle),
+    return Scaffold(
+      body: Container(
+        height: MediaQuery.of(context).size.height,
+        width: MediaQuery.of(context).size.width,
```

Comment:
use Get.width and Get.height when possible

Reply:
Done